libs/collectives/CMakeLists.txt

```diff
@@ -19,9 +19,15 @@ set(collectives_sources
 
 include(HPX_AddModule)
 add_hpx_module(collectives
+    FORCE_LINKING_GEN
```

Comment:
`FORCE_LINKING_GEN` is passed twice now (the other one is just below `DEPENDENCIES`).